src/send/SendConfirmation.tsx

```diff
@@ -174,124 +162,100 @@ function SendConfirmation(props: Props) {
-            <ContactCircle recipient={recipient} />
-            <View style={styles.recipientInfoContainer}>
-              <Text style={styles.headerText} testID="HeaderText">
-                {t('sending')}
-              </Text>
-              <Text testID="DisplayName" style={styles.displayName}>
-                {getDisplayName(recipient, t)}
-              </Text>
-              {showAddress && (
-                <View style={styles.addressContainer} testID="RecipientAddress">
-                  <ShortenedAddress style={styles.address} address={recipient.address} />
-                </View>
-              )}
-            </View>
-          </View>
-          <TokenDisplay
-            testID="SendAmount"
-            style={styles.amount}
-            amount={tokenAmount}
-            tokenId={tokenId}
-            showLocalAmount={false}
+      <ReviewContent>
+        <ReviewSummary>
+          <ReviewSummaryItem
+            testID="SendConfirmationToken"
+            header="Sending"
+            icon={<TokenIcon token={tokenInfo} />}
+            title={getDisplayTokenAmount(tokenAmount, tokenInfo!)}
+            subtitle={getDisplayLocalAmount(localAmount, localCurrencySymbol)}
+          />
+
+          <ReviewSummaryItemContact
+            testID="SendConfirmationRecipient"
+            header="To"
+            recipient={recipient}
+          />
+        </ReviewSummary>
+
+        <ReviewDetails>
+          <ReviewDetailsItem
+            testID="SendConfirmationNetwork"
+            label={t('transactionDetails.network')}
+            value={tokenInfo && NETWORK_NAMES[tokenInfo.networkId]}
+          />
+          <ReviewDetailsItem
+            testID="SendConfirmationFee"
+            label={t('networkFee')}
+            isLoading={prepareTransactionLoading}
+            value={
+              <Trans
+                i18nKey={'reviewTransaction.fullAmountApprox'}
+                context={fees.localAmount?.gt(0) ? undefined : 'noFiatPrice'}
+                values={{
+                  tokenAmount: fees.displayTokenAmount,
+                  localAmount: fees.displayLocalAmount,
+                  tokenSymbol: fees.tokenSymbol,
+                  localCurrencySymbol,
+                }}
+              >
+                <Text />
+              </Trans>
+            }
           />
-          <TokenDisplay
-            testID="SendAmountFiat"
-            style={styles.amountSubscript}
-            amount={tokenAmount}
-            tokenId={tokenInfo?.tokenId}
-            showLocalAmount={true}
+          <ReviewDetailsItem
+            testID="SendConfirmationTotal"
+            variant="bold"
+            label={t('reviewTransaction.totalPlusFees')}
+            isLoading={prepareTransactionLoading}
+            value={
+              <Trans
+                i18nKey={'reviewTransaction.fullAmountApprox'}
+                context={fees.localAmount?.gt(0) ? undefined : 'noFiatPrice'}
```

Comment:
```suggestion
                context={fees.totalDisplayLocalAmount?.gt(0) ? undefined : 'noFiatPrice'}
```
in case the send token does not have a fiat price but the fee token does

Reply:
It will be new `totalTokenAmount` field as `totalDisplayLocalAmount` in an already formatted string.